.github/workflows/release.yml

```diff
@@ -39,5 +49,26 @@ jobs:
             - name: Set access token
               run: yarn config set npmAuthToken "${{ secrets.NPM_AUTH_TOKEN }}"
 
-            - name: publish to npm
+            # It uses the Changesets GitHub Action to automate versioning and releases.
+            - name: Create Release Pull Request
+              uses: changesets/action@v1
+              with:
+                  # Executes the `changeset version` command. This will bump package versions
+                  # and update changelogs based on changesets, and include these changes in a new PR.
+                  version: yarn changeset version
+
+                  # Executes the `changeset tag` command to create a new version tag for each updated
+                  # package. These tags will be included in the new PR and will be applied when the PR is merged.
+                  publish: yarn changeset tag
```

Comment:
We currently sync the package versions across all packages, just to make updating simpler. 

Does this do the same? or do we need to move to a individual release model?

Reply:
(I don't mind changing to individual releases, I just want to be clear on what this is doing)

Reply:
If we need to sync the package versions across all packages, then changesets may not be the right tool for the requirement.